### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by huanghao1998**
What's change
- Add hover effect on the person list
- Able to unselect when the same person clicked
- Adjust the tag such that it can rearrange to the next line when the size of the panel is too small
- Update the latest help information

### CODE REVIEW

File: src/main/java/seedu/socket/ui/HelpWindow.java [hunk 1]
```diff
@@ -28,13 +28,24 @@ public class HelpWindow extends UiPart<Stage> {
...
             + "Format: edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] "
             + "[g/GITHUBPROFILE] [l/LANGUAGE] [t/TAG]…\n\n"
-            + "4. Locating persons by name or tag: find\n"
+            + "4. Removing a person's attribute : remove\n"
+            + "Format: remove INDEX [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] \"\n"
+            + "[g/GITHUBPROFILE] [l/LANGUAGE] [t/TAG]…\n\n"
+            + "5. Clearing groups of person : clear\n"
+            + "Format: clear [t/TAG]...\n\n"
+            + "6. Locating persons by name or tag: find\n"
             + "Format: find KEYWORD [MORE_KEYWORDS]… [t/TAG]…\n\n"
```
Comment (nus-jeff) [inline src/main/java/seedu/socket/ui/HelpWindow.java:37]: Ah I've forgotten to update the find command here... I'll update on my end after I merge.

**Review by nus-jeff — COMMENTED** [review]
LGTM, do check the comments~ thanks.

File: src/main/java/seedu/socket/ui/HelpWindow.java [hunk 2]
```diff
@@ -28,13 +28,24 @@ public class HelpWindow extends UiPart<Stage> {
             + "3. Editing a person : edit\n"
             + "Format: edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] "
             + "[g/GITHUBPROFILE] [l/LANGUAGE] [t/TAG]…\n\n"
-            + "4. Locating persons by name or tag: find\n"
+            + "4. Removing a person's attribute : remove\n"
+            + "Format: remove INDEX [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] \"\n"
```
[the remark this answers is not in the document]
Comment (huanghao1998) [reply src/main/java/seedu/socket/ui/HelpWindow.java:32]: Opps, good spot! I mistype the "

**Review by nus-jeff — APPROVED** [review]
LGTM 👍 thanks.

**Pull request merged**